src/content/eips/index.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: Ethereum Improvement Proposals (EIPs)
-description: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+title: Ethereum Improvement Proposals
+description: The basic information you need to understand EIPs
 lang: en
 sidebar: true
 ---
 
-# Introduction to Ethereum Improvement Proposals (EIPs) {#introduction-to-ethereum-improvement-proposals-eips}
+# Introduction to Ethereum Improvement Proposals {#introduction-to-ethereum-improvement-proposals}
```

Comment:
We'd want to keep this as per our style guide: https://ethereum.org/en/contributing/style-guide/#acronyms

```suggestion
# Introduction to Ethereum Improvement Proposals (EIPs) {#introduction-to-ethereum-improvement-proposals}
```